### DISCUSSION TITLE
How to get sound on the silence branch

### DISCUSSION THREAD

**Discussion by NackSunao** [dnschneid/crouton#5022] [General]
Hello, good morning everyone. I have managed to get sound working inside bullseye using the silence branch. I am not 100% sure how to do it, but I have done it four times now. In my case, all I need are wired headphones and a few simple commands.

Something to clarify is that I am not an expert in Linux or all of its functions, so I am not completely sure if a step is crucial or not.

Very well, in my case I have installed Debian Bullseye from VT-2 in response to the fact that sudo can no longer be used as before. I made a common installation with the silence branch using the following commands: sudo CROUTON_BRANCH=silence crouton -r bullseye -t xfce. 

Before you start with the cool stuff, you need to know that in my case I use the Citra appimage for this to work. I'm not sure if it's a requirement, but in my case it helps a lot. So, you start the appimage, go to settings, audio, select cubeb in output type and input type. Output device and input device are for later. Once you've done that, open a terminal and write the command: pulseaudio --start. That's it for now.

Now, you should log out of xfce or the desktop you use, end session in crosh and also in VT-2. After closing all sections, you should log back into your chroot, start the Citra appimage again, go to settings, audio and stay there, open a terminal and write again: pulseaudio --start, go to the sound symbol in the upper right corner and with that wala you should have your headphones selected in output devices in the volume control. I will attach test images of how it looks in my case.

![Screenshot_2023-10-16_15-45-22](https://github.com/dnschneid/crouton/assets/129456424/1dd171e2-c023-4c7e-b15f-e994edbad0fc)

**Comment by CroutonIsFun** [comment]
Wow. That's convoluted. I've never heard of Citra, but there's lots of stuff out there. I got sound working by purchasing an inexpensive USB audio adapter made by a company called Sabrent. Check Amazon for a USB audio adapter. There's a whole discussion page I wrote for this.

**Reply by DennisLfromGA** [reply]
Note to others:
The 'whole Discussion page' mentioned above is linked [here](https://github.com/dnschneid/crouton/discussions/4969).

-**DennisLfromGA**

**Reply by CroutonIsFun** [reply]
Thanks, Dennis. I should have included this in my reply.
